openxla_patches/thread_local_random.diff

```diff
@@ -0,0 +1,40 @@
+https://github.com/pytorch/xla/issues/4084
```

Comment:
if I remember correctly this is a XRT related patch we can just remove...

Reply:
thanks for pointing, test in https://github.com/pytorch/xla/pull/5221

Reply:
#5221's change has been saved in https://github.com/pytorch/xla/pull/5222 too, since https://github.com/pytorch/xla/pull/5222 has passed all tests, will apply this change in #5221

Reply:
Thanks, updated